gobblin-yarn/src/main/java/org/apache/gobblin/yarn/YarnHelixUtils.java

```diff
@@ -133,4 +131,8 @@ public static void addFileAsLocalResource(FileSystem fs, Path destFilePath, Loca
 
     return environmentVariableMap;
   }
+
+  public static String getContainerNum(String containerId) {
+    return "container-" + containerId.substring(containerId.lastIndexOf("_") + 1);
```

Comment:
why are we doing + 1?

Reply:
We want the substring starting from the char immediately following the last "_". e.g. if containerId = "container_e94_1567552810874_2132400_01_000001", we want to return 
"container-000001". Added javadoc to make the behavior clear.